backend/server/server_utils.py

```
@@ -174,6 +181,76 @@ async def handle_human_feedback(data: str):
+            "type": "chat",
+            "content": response_content,
+            "role": "assistant",
+            "metadata": {
+                "tool_calls": tool_calls_metadata
+            } if tool_calls_metadata else None
+        })
+        
+        logger.info(f"Chat response sent successfully")
+        
+    except json.JSONDecodeError as e:
+        logger.error(f"Failed to parse chat data: {e}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)